src/setup.py

```diff
@@ -455,6 +571,37 @@ def _configure_plugin_installation_method(self):
         elif answer == "3":
             self.config.use_recommended_collectd_config = True
 
+    def _configure_plugin_installation_method_non_interactive(self):
+        # recommended|add|not_modify
+        if not self.installation_method:
+            pass
+        else:
+            if self.installation_method == 'not_modify':
+                pass
+            if self.installation_method == 'add':
+                self.config.only_add_plugin = True
+            if self.installation_method == 'recommended':
+                self.config.use_recommended_collectd_config = True
+
+    def _configure_debug_non_interactive(self):
+            if self.debug:
+                self.config.debug = True
+
+    def _debug_setup(self):
+        logger.info('-=**********DEBUG**********=-')
+        logger.info('PUSH ASG: {}'.format(self.config.push_asg))
+        logger.info('PUSH CONSTANT: {}'.format(self.config.push_constant))
+        logger.info('CONSTANT DIMENSION VALUE: {}'.format(self.config.constant_dimension_value))
+        logger.info('SECRET KEY: {}'.format(self.config.secret_key))
+        logger.info('ACCESS KEY: {}'.format(self.config.access_key))
+        logger.info('HOST: {}'.format(self.config.host))
+        logger.info('PROXY NAME: {}'.format(self.config.proxy_server_name))
+        logger.info('PROXY PORT: {}'.format(self.config.proxy_server_port))
+        logger.info('CREDENTIALS PATH: {}'.format(self.config.credentials_path))
+        logger.info('METHOD ONLY ADD PLUGIN: {}'.format(self.config.only_add_plugin))
+        logger.info('USE RECOMMENDED CONFIG: {}'.format(self.config.use_recommended_collectd_config))
+        logger.info('REGION: {}'.format(self.config.region))
+        logger.info('IAM ROLE (metadata): {}'.format(self.metadata_reader.get_iam_role_name()))
```

Comment:
Could you add enable_high_resolution_metrics and flush_interval_in_seconds parameters output here?

Reply:
Done.

Reply:
Sorry, I tested with "./setup.py -I -e ", as "e" is a boolean parameter, it should be the same as push_asg,  could you fix it?

```
[root@ip-172-31-24-132 ~]# ./setup.py -I -e 
usage: a.py [-h] [-I] [-H HOST_NAME] [-r REGION] [-n NAME] [-p PORT]
            [-e ENABLE_HIGH_RESOLUTION_METRICS] [-f FLUSH_INTERVAL]
            [-a ACCESS_KEY] [-s SECRET_KEY] [-P CREDENTIALS_PATH]
            [-m recommended|add|not_modify] [-g] [-c] [-v DIMENSION_VALUE]
            [-d] [-D]

```